### PR TITLE
Favicon changes

### PR DESCRIPTION
**Pull request by DrSkyfaR**
+ patch-1

### CODE REVIEW

**Comment by mobybit** [comment]
Hi ApoRes1337,
at first: thank you very much for your pull request!

I think (please correct me of I'm wrong) in most cases it will be (technically) enough to place the favicon.ico in the root directory of the website. At the other side Natrium should follow the recommendations of [W3C](https://www.w3.org/2005/10/howto-favicon).

We should use only `icon` instead of `shortcut icon`. I also don't fully understand why you insert the following code: 

```
{{ else }} 
   href="/img/fav.ico" 
{{ end }}
```

In other words: I need an explaination ;) Thanks!

Regards,
Tim
